voice/bxml/verbs/transferConference.md

```diff
@@ -0,0 +1,90 @@
+{% method %}
+
+## XML: `<Transfer>`
+The Transfer verb with `<Conference>` is used to join the current call into a conference.
+
+### Attributes
+| Attribute | Description |
+|:----------|:------------|
+| None      | None        |
+
+### Nested Tags
+The nested tag `<Conference>` defines the conference the call will join.
+
+| Verb        | Description                                                                                                                     |
+|:------------|:--------------------------------------------------------------------------------------------------------------------------------|
+| Conference  | The name of the conference with alphanumeric characters and the symbols `-`, `_`, and `.` with maximum length of 50 characters. |
```

Comment:
I don't know if it is clear that we are expecting a `friendlyName`. 
Should we do the same as we do in `playAudio.md`?